Decred Wallet/Features/Navigation Menu/NavigationMenuTabBarController.swift

```diff
@@ -102,12 +102,11 @@ class NavigationMenuTabBarController: UITabBarController {
     
     static func setupMenuAndLaunchApp(isNewWallet: Bool) {
         // wallet is open, setup sync listener and start notification listener
```

Comment:
This comments needs to be updated.
Also move these 2 lines to the end of the function, after the root VC is set and displayed.

Reply:
resoolved in 3f5a019